pkg/asset/ignition/content/doc.go

```diff
@@ -1,3 +0,0 @@
-// Package content contains the contents of files and systemd units to be added
-// to Ignition configs.
```

Comment:
We'll want to keep this in the new package (vs. removing it completely, like you do here) to keep golint happy.

```console
$ git describe --dirty
v0.1.0-105-ge8d4c5a
$ golint -min_confidence 0.2 pkg/asset/ignition/bootstrap/content/
pkg/asset/ignition/bootstrap/content/bootkube.go:1:1: should have a package comment, unless it's in another file for this package
pkg/asset/ignition/bootstrap/content/kubelet.go:1:1: should have a package comment, unless it's in another file for this package
pkg/asset/ignition/bootstrap/content/report.go:1:1: should have a package comment, unless it's in another file for this package
pkg/asset/ignition/bootstrap/content/tectonic.go:1:1: should have a package comment, unless it's in another file for this package
```